### DISCUSSION TITLE
nvm windows

### DISCUSSION THREAD

**Discussion by wahid2511** [coreybutler/nvm-windows#1050] [Support] [answered]
how can I run different versions of node at the same time, for example in first project nvm use 14 and in the second project nvm use 16, in my case when I use nvm 14 in the first project then in the second project nvm is used 14 too , unlike Linux or MacOS, both OS can run two different node versions at the same time

**Comment by coreybutler** [comment, marked as answer]
This is answered in the [common issues](https://github.com/coreybutler/nvm-windows/wiki/Common-Issues#how-can-i-use-two-different-versions-of-nodejs-at-the-same-time) wiki page.
